### PR TITLE
Minor fix typos

### PR DESCRIPTION
**Pull request by ScorpioCPH**
Minor fix typos, @guineveresaenger @cblecker @idvoretskyi @stp-ip PTAL, Thanks!

/sig contributor-experience

### CODE REVIEW

File: contributors/guide/README.md [hunk 1]
```diff
@@ -1,9 +1,9 @@
 **OWNER:**
 
-sid-contributor-experience
+sig-contributor-experience
 
 ## Disclaimer
-Hello! This is the starting point for our brand new contributor guide, currently underway as per [-issue#6102](https://github.com/kubernetes/website/issues/6102) and in need of help. Please be patient, or fix a section below that needs improvement, and submit a pull request!
+Hello! This is the starting point for our brand new contributor guide, currently underway as previous [issue#6102](https://github.com/kubernetes/website/issues/6102) and in need of help. Please be patient, or fix a section below that needs improvement, and submit a pull request!
```
Comment (stp-ip) [inline contributors/guide/README.md]: "as per" was right in my opinion.

File: contributors/guide/README.md [hunk 2]
```diff
@@ -1,9 +1,9 @@
 **OWNER:**
 
-sid-contributor-experience
+sig-contributor-experience
```
Comment (stp-ip) [inline contributors/guide/README.md]: good catch

File: contributors/guide/README.md [hunk 3]
```diff
@@ -94,7 +94,7 @@ _Improvements needed_
 
 ### Find a SIG that is related to your contribution
 
-You may have noticed that some repositories in the Kubernetes Organizaion are owned by Special Interest Groups, or SIGs. We organize the Kubernetes community into SIGs in order to improve our workflow and more easily manage what is a very large community project.
+You may have noticed that some repositories in the Kubernetes Organization are owned by Special Interest Groups, or SIGs. We organize the Kubernetes community into SIGs in order to improve our workflow and more easily manage what is a very large community project.
```
Comment (stp-ip) [inline contributors/guide/README.md]: +1

File: contributors/guide/README.md (hunk 1 above)
Comment (ScorpioCPH) [reply contributors/guide/README.md]: Thanks, I missed `as`.

**Pull request merged**